frontend/.env.example

```diff
@@ -1,8 +0,0 @@
-REACT_APP_API_PORT=8000
-
-REACT_APP_FIREBASE_API_KEY=""
-REACT_APP_FIREBASE_AUTH_DOMAIN=""
-REACT_APP_FIREBASE_PROJECT_ID=""
-REACT_APP_FIREBASE_STORAGE_BUCKET=""
-REACT_APP_FIREBASE_MESSAGING_SENDER_ID=""
-REACT_APP_FIREBASE_APP_ID=""
```

Comment:
Could you please return this file?
You can remove variables that are no longer used